hs_core/autocomplete_light_registry.py

```diff
@@ -5,6 +5,8 @@ class UserAutocomplete(autocomplete_light.AutocompleteModelBase):
     search_fields=['username','email','first_name','last_name']
     
     def choice_label(self, choice):
+        if (choice.first_name + choice.last_name) and choice.email:
+            return choice.first_name + " " + choice.last_name + " - " + choice.email
```

Comment:
String concatenations are inefficient.

- Avoid them in the conditional
- Use string formatting in this return expression

Also, what if they have first or last name, but not both?